.github/workflows/ci.yml

```diff
@@ -29,12 +29,7 @@ jobs:
       with:
         base_sha: ${{ steps.last_successful_commit_push.outputs.commit_hash }}
         files: |
-          contrib/**
```

Comment:
For builds on the main branch, we only want to look at changes to the plugins/* directory. We don't want to trigger a push of all plugins if tests or Makefile change.